lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -372,6 +393,47 @@ putTx inputs outputs = do
         putAddress address
         putWord64be $ getCoin coin
 
+{-------------------------------------------------------------------------------
+                                 Stake Distribution
+-------------------------------------------------------------------------------}
+
+data StakeDistribution = StakeDistribution
+    { epoch :: Word64
+    , stake :: StakePools
+    } deriving (Show, Eq, Generic)
+
+-- | Coins are stored as Lovelace (reminder: 1 Lovelace = 1e-6 ADA)
+newtype Stake = Stake { getStake :: Word64 } deriving (Show, Eq)
+
+data StakePools = StakePools
+    { dangling :: Word64
+    , pools :: [(PoolId,Stake)]
+    , unassigned :: Word64
```

Comment:
Should the `Word64` of `dangling` and `unassigned` be `Stake` too?

Reply:
indeed, corrected!